### DISCUSSION TITLE
[Feature Request] Enhanced Email Signature

### DISCUSSION THREAD

**Discussion by mvroliv** [tutao/tutanota#3794] [Ideas]
Hello!

Request to implement more features for enhanced formatting on email signature, like custom HTML :)

**Comment by vitoreiji** [comment]
That's already possible: Settings > Email > Email signature > Custom.

**Comment by mvroliv** [comment]
I know of that possibility. I meant being able to have more formatting options :)
Also support for something like https://si.gnatu.re or similars.
